butterfree/configs/environment.py

```diff
@@ -11,6 +11,7 @@
     "FEATURE_STORE_HISTORICAL_DATABASE": "test",
     "KAFKA_CONSUMER_CONNECTION_STRING": "test_host:1234,test_host2:1234",
     "STREAM_CHECKPOINT_PATH": None,
+    "KAFKA_CONNECTION_STRING": "test",
```

Comment:
Don't need to create a `KAFKA_CONNECTION_STRING`, because we have the variable `KAFKA_CONSUMER_CONNECTION_STRING` :smiley_cat: